src/gcp/cloudfunctionsv2.ts

```diff
@@ -18,6 +18,14 @@ const client = new Client({
 });
 
 export const PUBSUB_PUBLISH_EVENT = "google.cloud.pubsub.topic.v1.messagePublished";
+export const STORAGE_EVENTS = [
```

Comment:
Should we make a distinction that these are only v2 storage events?

Reply:
Also, I'm wondering if we should export these from `services/index.ts` instead. We already have the event/service mapping there, so I can see us also defining the constants in that file.

Reply:
Since the events are defined in `src/gcp/cloudfunctionsv2.ts`, I felt that annotating w/ `_V2_` redundant.

In hindsight, I like your original structure where we had explicit module for keeping event types. Let me see what it looks like if I follow that model.